introduction/quickstart.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Installation
+layout: default
+nav_order: 1
+parent: Getting Started
+---
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Quickstart with SQL using the GlareDB CLI
+
+1. Install and start the GlareDB binary by running the following in a shell:
+
+   ```shell
+   curl https://glaredb.com/install.sh | sh
+
+   ./glaredb
+   ```
+
+2. Start writing SQL! You can try out the below to query a Parquet file hosted
+   on GitHub over HTTP. This file contains NYC real estate transactions from
+   January 2022.
+
+   ```sql
+   SELECT *
+   FROM read_parquet('https://github.com/GlareDB/tutorial_data/raw/main/nyc_sales/sale_year=2022/sale_month=1/f55363e2587849bcb25c057be706c69d-0.parquet');
+   ```
```

Comment:
given that we own the files, it might be nice to have shorter paths and more meaningful file names (rather than dipping into a hive-partition.